selenoid.go

```diff
@@ -427,12 +433,23 @@ func processBody(input []byte, host string) ([]byte, string, error) {
 				if raw, ok := v["capabilities"]; ok {
 					if c, ok := raw.(map[string]interface{}); ok {
 						sessionId = v["sessionId"].(string)
+
 						c["se:cdp"] = fmt.Sprintf("ws://%s/devtools/%s/", host, sessionId)
 						if rbv, ok := c["browserVersion"]; ok {
 							if bv, ok := rbv.(string); ok {
 								c["se:cdpVersion"] = bv
 							}
 						}
+
+						if c["webSocketUrl"] != nil {
+							u, err := url.Parse(c["webSocketUrl"].(string))
+							if err != nil {
+								return nil, sessionId, fmt.Errorf("parse 'websocketUrl' from response: %v", err)
+							} else {
+								u.Host = host
```

Comment:
Browser returns url with its own host which should be changed on selenoid host (like for `se:cdp`)